application/common/models/Reset.php

```diff
@@ -229,40 +250,37 @@ public function sendMethod()
      * @param string|null $ccAddress
      * @throws \Exception
      */
-    public function sendEmail($toAddress, $subject, $view, $ccAddress = null)
+    private function sendEmail($toAddress, $subject, $view, $ccAddress = null)
     {
         /*
          * Generate code if needed, update attempt counter, save record, and send email
          */
         if ($this->code === null) {
             $this->code = Utils::getRandomDigits(\Yii::$app->params['reset']['codeLength']);
         }
-        $this->attempts += 1;
-        if ($this->save()) {
-            // Send email verification
-            Verification::sendEmail(
-                $toAddress,
-                $subject,
-                '@common/mail/reset/' . $view,
-                $this->code,
-                $this->user,
-                $ccAddress,
-                $this->user->id,
-                self::TOPIC_RESET_EMAIL_SENT,
-                'Password reset email for ' . $this->user->getDisplayName() .
-                'sent to ' . $toAddress
-            );
 
-        } else {
-            throw new \Exception('Unable to update reset in database, email not sent', 1461098651);
-        }
+        $this->saveOrError('send email','Unable to update reset in database, email not sent.');
+
+        // Send email verification
+        Verification::sendEmail(
+            $toAddress,
+            $subject,
+            '@common/mail/reset/' . $view,
+            $this->code,
+            $this->user,
+            $ccAddress,
+            $this->user->id,
+            self::TOPIC_RESET_EMAIL_SENT,
+            'Password reset email for ' . $this->user->getDisplayName() .
+            'sent to ' . $toAddress
```

Comment:
Looks like you may need another space here: `' sent`.
Similarly below in `sendPhone()`.